Workbooks/Lighthouse/Overview/Overview.workbook

```diff
@@ -0,0 +1,343 @@
+                }
+              ],
+              "allowCustom": true
+            }
+          },
+          {
+            "id": "c757ee66-876f-4c53-b6bc-50c19fc9e227",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resource",
+            "type": 5,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "summarize by type\r\n| where type <> \"\"\r\n| project Resource = toupper(substring(type,0,indexof( type, '/')))\r\n| distinct Resource",
+            "crossComponentResources": [
+              "{Subscription}"
+            ],
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "selectAllValue": "All"
+            },
+            "timeContext": {
+              "durationMs": 2592000000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "02c70ae9-f0b4-425d-b429-37e1df1edb37",
+            "version": "KqlParameterItem/1.0",
+            "name": "User",
+            "label": "User Contains",
+            "type": 2,
+            "description": "This will filter callers with this string in their name",
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "AzureActivity\r\n| where Caller contains '@'\r\n| summarize by Caller",
+            "crossComponentResources": [
+              "{Workspace}"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "selectAllValue": "All"
+            },
+            "timeContext": {
+              "durationMs": 2592000000
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces",
+            "value": [
+              "value::all"
+            ]
+          }
+        ],
+        "style": "pills",
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "name": "parameters - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AzureActivity\r\n//| where Caller contains ({User})\r\n| where \"{User:lable}\" == \"All\" or Caller in ({User})\r\n| where Caller contains '@'\r\n| where ActivityStatusValue  == \"Success\" \r\n| where ResourceProviderValue <> \"\"\r\n| where \"{Resource:label}\" == \"All\" or ResourceProviderValue in ({Resource})\r\n| summarize Deletes = countif(OperationNameValue contains \"DELETE\"), Writes = countif(OperationNameValue contains \"WRITE\"),  Activities = (countif(OperationNameValue contains \"DELETE\") + countif(OperationNameValue contains \"WRITE\")) by bin_at(TimeGenerated, 1h, now())\r\n",
```

Comment:
`{user:lable}` is a mispelling?

Reply:
actually, your check here of `label == "all"` will only work in english, and will not work in other languages.

see https://github.com/microsoft/Application-Insights-Workbooks/blob/jgardner/missingTableHint/Documentation/Parameters/DropDown.md#special-casing-all for a better way to do this